ImportExport/Export2Slicer.FCMacro

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
```

Comment:
Sorry about this but now I find `ExportToSlicer.FCMacro` a better name. Is it OK to change it?